src/builtins/list.zig

```diff
@@ -25,10 +20,11 @@ const IncN = *const fn (?[*]u8, usize) callconv(.C) void;
 const Dec = *const fn (?[*]u8) callconv(.C) void;
 const HasTagId = *const fn (u16, ?[*]u8) callconv(.C) extern struct { matched: bool, data: ?[*]u8 };
 
-const SEAMLESS_SLICE_BIT: usize =
+/// TODO
```

Comment:
```suggestion
/// A bit mask were the only set bit is the bit indicating if the List is a seamless slice.
```

Reply:
Thank you, added in commit 8151d4768590c9559588bd99c0201f8ac60613c3